package.json

```diff
@@ -93,5 +94,13 @@
     "node-sass": "^4.11.0",
     "redux-mock-store": "^1.5.3",
     "sass-lint": "^1.12.1"
+  },
+  "peerDependencies": {
+    "@entando/apimanager": "^3.0.2",
+    "@entando/messages": "^1.0.4",
+    "react-router": "^4.3.1",
+    "react-router-dom": "^4.3.1",
```

Comment:
no reason to install react-router on top of dom, and you should be installing just version 5 not 4

Reply:
Hi @nicpuddu. I'll make changes to that. Also, I'll be adding more modules to peer dependencies

Reply:
hi @nicpuddu, I have already changed `react-router-dom` to v5 and removed `react-router` as well. I have also added more peer dependencies.